_posts/2019-08-06-pre-post-events-sustainable-trains.md

```diff
@@ -0,0 +1,114 @@
+|---------------------------------|-----------|-----------|---------|-------------------------------------------------------------------------------------------|
+| Gothenburg, SE                  | Wednesday | 08:55     | 12:28   | Swing by the office before and grab a laptop and a coffee to go.                          |
+| Kopenhagen, DK                  | Wednesday | 12:52     | 18:02   | Via Fredericia st.                                                                        |
+| Hamburg, DE                     | Wednesday | 18:02     |         | Here we could have a meetup in Hamburg, probably best with food.                          |
+| Hamburg, DE                     | Wednesday | 21:02     | 07:54   | Now we can fade out the day and rest.                                                     |
+| Basel, CH                       | Thursday  | 08:07     | 09:00   | This is the high speed train, you can stay on the NightJet and arrive a little later.      |
+| Zürich HB, CH                   | Thursday  |           | 09:42   | Time to walk a little in Zürich, meet people and maybe hit a shower and a day of hacking. |
+| Zürich HB, CH                   | Thursday  | 12:03     | 14:45   | At this point our group might have grown a bit.                                           |
+| Genève, CH                      | Thursday  | 14:59     | 18:05   | The fast TGV. 🚅                                                                          |
+| Valence TGV Rhône-Alpes Sud, FR | Thursday  | 18:22     | 22:44   | Say goodbye to the Alps.                                                                  |
+| Barcelona Sants, ES             | Thursday  |           | 22:44   | 🎉 Hurray, we made it.                                                                       |
+
+
+### From the East
+
+Now what if we were able to meet up with people?
+Lucky us there is a direct line from Budapest to Zürich.
+Let's look at the connection:
+
+{:.layout-auto}
+| City                            | Day       | Departure | Arrival | Notes                                                                                   |
+|---------------------------------|-----------|-----------|---------|-----------------------------------------------------------------------------------------|
+| Budapest-Keleti, HU             | Wednesday | 20:40     | 08:20   | Swing by the office before and grab a laptop and a coffee to go.                        |
+| Vienna, AT                      | Wednesday | 23:25     | 08:20   | Same train, has multiple stops in Austria.                                              |
+| Zürich HB, CH                   | Thursday  |           | 08:20   | Time to walk a little in Zürich, meet people and maybe hit a shower and day of hacking. |
+
+Note that you can take a direct train from Warsaw/Warszawa or Prague/Praha to Vienna/Wien.
+
```

Comment:
How do people coordinate? Is there a contact point? Maybe make it clear that people need to self-organize (and buy their tickets!)

Can you link to the website with something like "We will keep updated information about routes and travel groups on our website"

Reply:
Our website and the rust users forum?